.circleci/config.yml

```diff
@@ -73,9 +73,13 @@ jobs:
       - run:
           name: Download prebuilt static node binary
           command: |
-            apk add wget
-            wget https://github.com/codecov/node-static-alpine/releases/download/node.v14.16.1_0b668b007c0533fa4be1b24c66941a06f5965fdf/node
-
+            apk add wget coreutils
+            NODE_RELEASE=node.v14.16.1_0b668b007c0533fa4be1b24c66941a06f5965fdf
+            wget https://github.com/codecov/node-static-alpine/releases/download/$NODE_RELEASE/node
+            wget https://github.com/codecov/node-static-alpine/releases/download/$NODE_RELEASE/SHA1SUM
+            wget https://github.com/codecov/node-static-alpine/releases/download/$NODE_RELEASE/SHA256SUM
+            wget https://github.com/codecov/node-static-alpine/releases/download/$NODE_RELEASE/SHA512SUM
+            for i in 1 256 512; do "sha${i}sum" node -c "SHA${i}SUM"; done
```

Comment:
```suggestion
            for i in 1 256 512; do "shasum" node -c "SHA${i}SUM"; done
```

maybe?

Reply:
Sadly, no. https://pkgs.alpinelinux.org/contents?branch=v3.3&name=coreutils&arch=x86&repo=main There is no `shasum` in apline. 😒 

It seems annoyed by the file, not the command.

Reply:
is the format maybe wrong? we are doing
```
sha1sum -c --ignore-missing <(curl -s https://raw.githubusercontent.com/codecov/codecov-bash/${VERSION}/SHA1SUM)
```
in the Codecov Action. Can you `cat` the file contents?

Reply:
i.e. why is `node` an argument